lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -454,6 +494,7 @@ export abstract class ClientApplication {
      * @param {@link (EndSessionRequest:type)} 
      */
     async logout(logoutRequest?: EndSessionRequest): Promise<void> {
+        this.broadcastEvent(BroadcastEvent.LOGOUT_START, null, logoutRequest);
```

Comment:
Should we wrap the `authClient.getLogoutUri(validLogoutRequest)` below in a try/catch that would broadcast `LOGOUT_SUCCESS` or `LOGOUT_FAILURE` event? Or would it be better to find a way to broadcast these events when we return from the server redirect?

Reply:
We added the try/catch. Still open to suggestions on how it could be done after the server redirect, but the try/catch works for now.